spec/support/features/doc_auth_helper.rb

```diff
@@ -109,6 +109,20 @@ def complete_doc_auth_steps_before_document_capture_step(expect_accessible: fals
     expect_page_to_have_no_accessibility_violations(page) if expect_accessible
   end
 
+  def complete_up_to_how_to_verify_step_for_opt_in_ipp_and_opting_in
```

Comment:
Can we make these into a single method called `complete_how_to_verify_step` that accepts a single argument, which is the path (remote or ipp) that the user is selecting?

Reply:
I like it.